projects/extension/ai/litellm.py

```diff
@@ -0,0 +1,35 @@
+import litellm
+from typing import Optional, Generator
+
+
+def embed(
+    model: str,
+    input: list[str],
+    api_key: str,
+    user: Optional[str] = None,
```

Comment:
Is the extension still python 3.9? Otherwise we should be able to use union `| None` syntax no?
Bit of a nit, just confused.

Reply:
We actually have a >= 3.10 in the pyproject.toml, so no. We have Optional[...] all over the place. I'd prefer that we address a change like that separately, and ensure that linting is configured correctly to enforce.